modules/psap-special-resource-operator.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="about-special-resource-operator"]
+= About the Special Resource Operator
+
+[discrete]
+== Background
+The Special Resource Operator (SRO) helps you manage the deployment of kernel modules and drivers on an existing (day 2) {product-title} cluster. SRO can be used for a case as simple as building and loading a single kernel module, or as complex as deploying the driver, device plugin, and monitoring stack for a hardware accelerator.
+
+For loading kernel modules, SRO is designed around the use of "driver containers." Driver containers are increasingly being used in cloud-native environments, especially when run on pure container operating systems to deliver hardware drivers to the host. Driver containers extend the kernel stack beyond the out-of-box software and hardware features of a specific kernel. Driver containers work on various container capable Linux distributions. With driver containers the host operating system stays "clean" and there will not be any clash between different library versions or binaries on the host.
```

Comment:
s/"driver containers."/driver containers./

I think the ones around "clean" should stay. (At least for now.)